modules/synacormediaBidAdapter.js

```diff
@@ -49,20 +57,33 @@ export const spec = {
         logWarn(`Synacormedia: there is an invalid POS: ${bid.params.pos}`);
         pos = 0;
       }
+      let videoOrBannerKey = this.isVideoBid(bid) ? 'video' : 'banner';
       getAdUnitSizes(bid).forEach((size, i) => {
-        let request = {
-          id: bid.bidId + '~' + size[0] + 'x' + size[1],
-          tagid: placementId,
-          banner: {
-            w: size[0],
-            h: size[1],
-            pos
-          }
+        if (!size || size.length != 2) {
+          return;
+        }
+        let size0 = size[0];
+        let size1 = size[1];
+        let imp = {
+          id: videoOrBannerKey.substring(0, 1) + bid.bidId + '-' + size0 + 'x' + size1,
```

Comment:
Use template strings if possible in Prebid, so line 68 is changed to something like: 
id: \``${videoOrBannerKey.substring(0,1)}${bid.bidId}-${size0}x${size1}`\`